src/main/java/com/agilethought/atsceapi/service/UserService.java

```diff
@@ -1,5 +1,9 @@
 package com.agilethought.atsceapi.service;
 
-public interface UserService {
+import java.util.List;
+
+import com.agilethought.atsceapi.model.UserModel;
 
+public interface UserService {
+	public List<UserModel> getAllUsers();

```

Comment:
`public ` keyword is not necessary due to `UserService ` is a interface, here we just define methods. Access modifiers are applied inside of implementation class.